modules/setup/library/Setup/WebWizard.php

```diff
@@ -576,6 +576,7 @@ public function getRequirements($skipModules = false)
         )));
 
         $set->add(new PhpConfigRequirement(array(
+            'optional'      => true,
```

Comment:
Please only do this for PHP versions lower than 7.0. v5.6 is still affected.

Reply:
I've fixed this in #3747. POC module:

```
[root@07b1f18d5654 /]# cat icingaweb2/modules/badtz/application/clicommands/ProofCommand.php
<?php

namespace Icinga\Module\Badtz\Clicommands;

use Icinga\Cli\Command;

class ProofCommand extends Command
{
    public function fallbackAction()
    {
        var_dump(@date_default_timezone_get());
        var_dump(date_default_timezone_get());
    }
}
[root@07b1f18d5654 /]# cat icingaweb2/modules/badtz/application/controllers/ProofController.php
<?php

namespace Icinga\Module\Badtz\Controllers;

use Icinga\Web\Controller;

class ProofController extends Controller
{
    protected $requiresAuthentication = false;

    public function fallbackAction()
    {
        var_dump(@date_default_timezone_get());
        var_dump(date_default_timezone_get());
    }
}
[root@07b1f18d5654 /]# cat icingaweb2/modules/badtz/application/views/scripts/proof/fallback.phtml
[root@07b1f18d5654 /]#
```